frontend/src/bindings/rust_ts_definitions.ts

```diff
@@ -2,19 +2,42 @@
  Generated by typeshare 1.0.0
```

Comment:
I would rename the file simply to definitions.ts (if it is generated by Rust doesn't matter and ts is already in file ending)